evnex/api.py

```diff
@@ -323,6 +324,25 @@ async def set_charge_point_override(
         r.raise_for_status()
         return True
 
+    @retry(
+        wait=wait_random_exponential(multiplier=1, max=60),
+        retry=retry_if_not_exception_type((ValidationError, NotAuthorizedException)),
+    )
+    async def get_charge_point_status(
+        self, charge_point_id: str
+    ) -> EvnexChargePointStatus:
+        """
+        :param charge_point_id:
+        :raises: ReadTimeout if the charge point is offline.
+        """
+        r = await self.httpx_client.post(
+            f"https://client-api.evnex.io/charge-points/{charge_point_id}/commands/get-status",
```

Comment:
I realize this just follows the convention I started but in hindsight the base url should be extracted

Reply:
Yup agreed, I'll open a different pr to fix that